app/views/patients/show.html.erb

```diff
@@ -1,3 +1,5 @@
+<% content_for :page_title, @patient.initials %>
```

Comment:
On the `h1` line below you can change the `page_title:` attribute rather than needing this line.

Reply:
done